Frameworks/include/CoreTextInternal.h

```diff
@@ -25,6 +25,14 @@
 #include <COMIncludes_End.h>
 #import <vector>
 
+template <typename T>
+inline void SafeRelease(T** p) {
```

Comment:
Renamed it _SafeRelease

---
In reply to: [78454639](https://github.com/Microsoft/WinObjC/pull/999#discussion_r78454639) [](ancestors = 78454639)